### ISSUE TITLE
Docs directory is looking a bit bare

### ISSUE THREAD

**Issue by jjl** [mpg-project/mpg#8]
At the minute, I'd say the README is doing a better job. It might be nice to include a few fully working basic examples in an 'examples' directory, but I don't think we can add much more value by having just markdown, given the "gets out of your way" nature of the library.

I think the best value from this module is given in conjunction with [utrecht](https://github.com/irresponsible/utrecht/) and this might make a nice base for samples.

Thoughts?

**Comment by jjl** [comment]
Oh another thought: It would be nice to ship the README in the artifacts. I spent a few weeks without internet access earlier this year and I wish all the modules I used shipped READMEs in their jars :)

**Comment by JuneKelly** [comment]
I'd agree with having more docs, particularly some examples, as I find them to be generally more useful than a page full of function doc-strings.

Maybe a one-page "Getting Started" walkthrough would be a good candidate to start?

**Comment by jjl** [comment]
Yeah, that sounds like a good idea. I'll ping you when my release manager has got the first release of utrecht out the door.

**Comment by jjl** [comment]
utrecht 0.2.0 hit clojars last night!
